app/controllers/participant_signup_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ParticipantSignupController < ApplicationController
+	before_filter :fetch_participant
+  	skip_before_filter :check_authorization	
```

Comment:
I gave some pretty bad/incomplete advice and suggested the `skip_before_filter` as the GET form route was being developed, but didn't give any recommendation about securing the PUT route.

Should have recommended an override instead.